influxdb2_client/src/models/label.rs

```diff
@@ -2,6 +2,88 @@
 
 use serde::{Deserialize, Serialize};
 
+/// Post create label request, to create a new label
+#[derive(Clone, Debug, PartialEq, Default, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct LabelCreateRequest {
+    /// Organisation ID
+    #[serde(rename = "orgID")]
+    pub org_id: String,
+    /// Organisation name
```

Comment:
I don't think this `name` parameter is the organization name, [I think it's the label name](https://docs.influxdata.com/influxdb/v2.0/api/#operation/PostLabels)? I could be wrong though.

```suggestion
    /// Label name
```

Reply:
It is Label name, I'll fix it